lib/logstash/filters/kv.rb

```diff
@@ -158,6 +158,20 @@ class LogStash::Filters::KV < LogStash::Filters::Base
   #     }
   config :default_keys, :validate => :hash, :default => {}
 
+  # A bool option for removing duplicate key/value pairs. When set to true, only 
+  # one key/value pair will exist, otherwise both will.
+  #
+  # For example, consider a source like `from=me from=me`. `[from]` will map to 
+  # an Array with two elements: `["me", "me"]`. to only keep unique key/value pairs,
+  # you could use this configuration:
+  # [source,ruby]
+  #     filter {
+  #       kv {
+  #         unique_values => true
+  #       }
+  #     }
+  config :unique_values, :validate => :boolean, :default => false
```

Comment:
hmm just feel `unique_values` may not actually reflect what we are trying to convey here..might get confused with having unique values across keys? don't know what the use case for that is, but just saying...

how about `allow_duplicate_values` ?